src/datascience-ui/ipywidgets/common/requirejsRegistry.ts

```diff
@@ -45,8 +45,8 @@ function getScriptsWithAValidScriptUriToBeRegistered(scripts: WidgetScriptSource
 
 function registerScriptsInRequireJs(scripts: NonPartial<WidgetScriptSource>[]) {
     // tslint:disable-next-line: no-any
-    const requirejs = (window as any).requirejs as { config: Function };
-    if (!requirejs) {
+    const requirejsfunc = (window as any).requirejs as { config: Function };
```

Comment:
Shouldn't this be `const requirejsfunc = (requirejs as any) || (window as any).requirejs;`

Reply:
Yeah you'd think so but this is never called or it's working as is.